src/App.js

```diff
@@ -0,0 +1,26 @@
+import  { useState, useEffect } from 'react';
+import User from './component/User';
+import './index.css';
+
+function App() {
+  const [users, setUsers] = useState([]);
+
+	useEffect(() => {
+		fetch('https://random-data-api.com/api/users/random_user?size=10')
+			.then(response => response.json())
+			.then(data => setUsers(data));
+	}, []);
+
+	return (
+	<div className='flex flex-col items-center justify-center bg-[#e1f6dc] h-full'>
+    	<button onClick={() => window.location.reload()} className="mt-8 bg-[#f0003c] rounded-lg px-16 py-2 text-2xl text-white">Reload Users</button>
```

Comment:
I don't think this is the right implementation we should go for on this button. According to the exercises instructions, The button should be `Fetch random`  where when clicked it should make another request to the API and bring a new set of users **without reloading the page**.